.github/workflows/go.yml

```diff
@@ -37,17 +55,30 @@ jobs:
         uses: actions/checkout@v2
       - name: Update dependencies 
         run: GOPROXY=https://proxy.golang.org go mod download
-      - name: Lint
-        run: make lint
-      - name: Check
-        run: make check
       - name: Build
         run: make
       - name: Build docker image
         run: make docker-build
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+        id: go
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: Update dependencies
+        run: GOPROXY=https://proxy.golang.org go mod download
+      - name: "install kubebuilder"
+        run: ./hack/install-kubebuilder.sh
+      - name: tests
+        run: make test
   checks:
-    name: build
```

Comment:
this job is to make the required check status pass, if you want to change the name, may need to also change the one in `.asf.yaml`

Reply:
done. change the name to `ci` that is more sense to me.

Reply:
The `build` job needs be kept for now to make checks status pass, after this gets merged (`.asf` takes effect`), we can remove it then